src/client/components/EditorWYSIWYG/EditorWYSIWYGLinks/EditorWYSIWYGLinks.tsx

```diff
@@ -10,6 +13,11 @@ type Props = {
 }
 const EditorWYSIWYGLinks: React.FC<Props> = (props: Props) => {
   const { onChange, value, disabled, repository } = props
+  const { print } = useIsPrintRoute()
+
+  if (print) {
+    return <EditorWYSIWYGReadOnly value={value} />
+  }
```

Comment:
Bonus fix:
- Introduce `EditorWYSIWYGReadOnly` and use it also for `EditorWYSIWYGLinks`